templates/magazine/panel/_options.html.twig

```diff
@@ -64,33 +65,4 @@
             </a>
         </li>
     </menu>
-    {% if is_route_name('magazine_panel_reports') %}
```

Comment:
this was unfunctional as far as I could tell. I didn't even notice it had existed before, but clicking the links does nothing

![image](https://github.com/MbinOrg/mbin/assets/146029455/e739f155-14aa-4d91-a43e-efd6252f6547)